### PR TITLE
24w36a

### PR DESCRIPTION
**Pull request by DragonWrangler1**

### CODE REVIEW

**Comment by DragonWrangler1** [comment]
I actually think I won't use MCLA's new effects api, However I will cherrypick a commit.

**Review by crispiebacon — CHANGES_REQUESTED** [review]

**Comment by crispiebacon** [comment]
It's much better than the other pr. And this time I reviewed all files... I do have one small question. How come there are some many files that it says are empty ?

**Comment by DragonWrangler1** [comment]
> It's much better than the other pr. And this time I reviewed all files.

I tried to fix all the things, next time I'll be more careful when using sed. I didn't think about the fact that I was in the main VoxelForge directory and not the EFFECTS directory.

> I do have one small question. How come there are some many files that it says are empty ?

Probably because I had updated them initially.

File: mods/CORE/vlf_textures/textures/PP/block/vlf_potions_effect_dolphin_grace.png
Comment (crispiebacon) [inline mods/CORE/vlf_textures/textures/PP/block/vlf_potions_effect_dolphin_grace.png:1]: These aren't supposed to be here correct ?

**Review by crispiebacon — APPROVED** [review]
Other than that one nitpick, Approving.

**Comment by DragonWrangler1** [comment]
@crispiebacon : Removed the textures.

**Comment by crispiebacon** [comment]
I can't seem to find any bugs.

**Pull request merged**